### PR TITLE
Suggested Terms in Channel Search

### PR DESCRIPTION
**Pull request by abeglova**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
Screen Shot 2020-04-03 at 3 41 58 PM
Screen Shot 2020-04-03 at 3 42 23 PM

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2721

#### What's this PR do?
This PR updates the code to display suggestions in the channel search when there are no hits for a search term, similar to the ones in the learning resource search

#### How should this be manually tested?
Create 5-6 comments and posts with the word "robot" 
Go to http://localhost:8063/search/ and search for "robo". Verify that "robot"  is returned as a suggestion and clicking the link searches for "robot" .

Also verify that the channel search still works for terms for which there are hits and that the learning resource search still works as expected as well

### CODE REVIEW

File: search/api_test.py [hunk 1]
```diff
@@ -64,6 +64,37 @@ def return_search_arg(
     )
 
 
+def raw_suggestions():
```
Comment (mbertrand) [inline search/api_test.py:67]: Minor nit - this doesn't need to be a function

File: static/js/lib/search.js [hunk 2]
```diff
@@ -173,7 +173,8 @@ const LIST_QUERY_FIELDS = [
   "topics"
 ]
 
-const SUGGEST_FIELDS = ["title", "short_description"]
+const LEARN_SUGGEST_FIELDS = ["title.trigram", "short_description.trigram"]
+const CHANNEL_SUGGEST_FIELDS = ["suggest_field1", "suggest_field2"]
```
Comment (mbertrand) [inline static/js/lib/search.js:177]: What do you think of adding `suggest_field_1` and `suggest_field_2` aliases to the `LEARNING_RESOURCE_TYPE` schema for consistency, and then you wouldn't need separate suggest field arrays for learning and channels?

Comment (abeglova) [reply static/js/lib/search.js:177]: I think that will cause data from learning resource fields to be used to generate suggestions for course_catalog fields and vice versa.

**Comment by abeglova** [comment]
Thanks @mbertrand. Back to you, I agree that the user receiving suggestions from channels they are not subscribed to isn't great. I also don't know how to fix that

**Pull request merged**